estimagic/tests/visualization/minimal_expected_df.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
A csv would be more readable as the nested structure is confusing.

Reply:
Yes! Implemented in 0846b2d5aa68a1880f930eade981bce6930216aa.